docs/Developers/Review.rst

```diff
@@ -3,19 +3,51 @@ Reviewing
 
 This document is a guide to coala's review process.
 
+Am I Good Enough to Do Code Review?
+-----------------------------------
+
+**Yes,** if you already fixed a newcomer issue.
+
+But be careful: **bad reviews will scare away other contributors.**
+
+You can and should help us review source code. Especially try to review source
+code of others and share what you have learnt with them. You can use acks and
+unacks like everyone else and ``cobot`` even allows you to set PRs to WIP.
+
+Generally follow this process:
+
+1. Check if the change is helping us. Sometimes people propose changes that are
+   only helpful for specific usecases but may break others. The concept has to
+   be good. Consider engaging in a discussion on gitter if you are unsure!
+2. Check for automatic builds. Give the contributor hints on how he can resolve
+   them.
+3. Review the actual code. Suggest improvements and simplifications.
+
+**Be sure to not value quantity over quality! Be transparent and polite:
```

Comment:
Bolding two statements is like writing if clauses in one line. Not as helpful and imidiately obvious as it should be. Consider to break these sentences up a little bit, maybe also adding the "try not to scare people away thing" somewhere around here.